### PR TITLE
feat(chat-service): Bots on the viewers list?

### LINKED ISSUE #29: Add feature to remove bots from viewers list
Hi guys, the PR #25 introduce one interessant feature to the project, but it doesn't merged because others problems, so I think to extract the feature and merge for the actual structure will be so cool.

### PR DESCRIPTION
**Pull request by omurilo**
Add feature to remove bots from viewers list to close #29

### CODE REVIEW

File: src/services/ChatterService.ts [hunk 1]
```diff
@@ -25,7 +35,14 @@ class ChatterService {
 
     const { viewers } = response.data.chatters;
 
-    return viewers;
+    return this.removeBots(viewers);
+  }
+
+  private async removeBots(viewers: Participants): Promise<Participants> {
```
Comment (lucasmontano) [inline src/services/ChatterService.ts:41]: I think we don't need to fetch the bots every time we fetch the current participants on `getViewers()`. Maybe we can load the bot list once at starting the service?

Comment (omurilo) [reply src/services/ChatterService.ts:41]: This is a good point, I don't thinked it.

But it is ease to fix ;D

**Comment by omurilo** [comment]
> Hey, i liked how you solved this compared to the first implementation. It's perfect. Can you also write new ones or refactor those tests please?

Because removeBots it is a private method i don't think on a test for him, but i've refactor the existing method to add a test to verify if this.bots is greather than or equal 0.

**Comment by powilliam** [comment]
There's no problems. Great job here!

**Comment by EduardoAraujoB** [comment]
do you know if this PR will have conflicts with the PR #33 ?

**Comment by nataneb32** [comment]
Nope. It doesn't.

**Comment by nataneb32** [comment]
Actually the only conflict is in the .env.example. This PR has a better version than mine. When merge it's the only thing to be cautious about.

**Pull request merged**